src/patternfly/components/Drawer/drawer.scss

```diff
@@ -173,6 +174,8 @@ $pf-v6-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   &.pf-m-inline,
   &.pf-m-static {
     > .#{$drawer}__main > .#{$drawer}__panel:not(.pf-m-no-border, .pf-m-resizable) {
+      --#{$drawer}__panel--BackgroundColor: var(--#{$drawer}__panel--m-inline--BackgroundColor);
+
       padding-inline-start: var(--#{$drawer}--m-inline__panel--PaddingInlineStart);
     }
```

Comment:
Adding it in this block will keep it from applying on no-border/resizeable drawers:

<img width="852" alt="Screenshot 2024-09-13 at 11 45 20 AM" src="https://github.com/user-attachments/assets/627ea680-c786-4866-9873-3e3a69355333">

<img width="860" alt="Screenshot 2024-09-13 at 11 46 07 AM" src="https://github.com/user-attachments/assets/9c912b34-3464-4876-bd24-acff5e0f9aa4">

I wonder if this update below would work?

```suggestion
  &.pf-m-inline,
  &.pf-m-static {
    --#{$drawer}__panel--BackgroundColor: var(--#{$drawer}__panel--m-inline--BackgroundColor);

    > .#{$drawer}__main > .#{$drawer}__panel:not(.pf-m-no-border, .pf-m-resizable) {
      padding-inline-start: var(--#{$drawer}--m-inline__panel--PaddingInlineStart);
    }
```